src/ui/ProcedureSpecEditor.as

```diff
@@ -49,6 +49,43 @@ public class ProcedureSpecEditor extends Sprite {
 	private const labelColor:int = 0x8738bf; // 0x6c36b3; // 0x9c35b3;
 	private const selectedLabelColor:int = 0xefa6ff;
 
+	// Probably out of date list of all %m and %d options
```

Comment:
Probably not. But you should still steal the menu codes from there so you don't miss any new ones!